.coderabbit.yaml

```
@@ -0,0 +1,4 @@
+reviews:
+  profile: assertive
+  sequence_diagrams: false
+  poem: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)